pyxem/utils/polar_transform_utils.py

```diff
@@ -160,7 +159,17 @@ def get_template_cartesian_coordinates(
     x = c * ox - s * oy + center[0]
     y = s * ox + c * oy + center[1]
     if window_size is not None:
-        condition = (x < window_size[0]) & (y < window_size[1]) & (y >= 0) & (x >= 0)
+        try:
```

Comment:
For me this has some code smells. Personally I would think it would be more clear to add a different argument, e.g. `limiting_radius`. Then write some boolean logic that determines the precedence of different "filtering" arguments

Reply:
Sure, that is probably clearer.

The reason I preferred this solution was that I don't have an overview of how other parts of pyxem that aren't neccesarily included in the tests use this function. Introducing an additional argument could break stuff I didn't want to tamper too much with.

Reply:
I'm not sure if any boolean logic is required to determine the precedence of the different "filtering arguments"? If a limiting radius is given that is smaller than the specified window, then you probably wouldn't want to include them anyway, and vica verca. Filtering the reflections first using a limiting radius and then filtering based on the window should do the trick, right?